SECURITY.md

```diff
@@ -0,0 +1,47 @@
+# Security Policy
+
+## Supported Versions
+
+| Version   | Supported          |
+| --------- | ------------------ |
+| 1.0.x     | :white_check_mark: |
+| 0.38.5    | :white_check_mark: |
+| <= 0.38.4 | :x:                |
+
+## Reporting Security Issues
+
+⚠ **Please do not report security vulnerabilities through public GitHub issues.** ⚠
+
+Instead, either encrypted through Keybase chat
+  - [keybase.io/cakebuild](https://keybase.io/cakebuild)
+
+Or report them to using NuGet.org *Contact owners*
```

Comment:
"Or report them ~**to**~ using NuGet.org *Contact owners*"

Tiny suggestion:
- "Or report them using NuGet.org *Contact owners*"

or
- "Or report them to **us** using NuGet.org *Contact owners*"

Reply:
Good feedback, all adjust to the latter 👍